contractsV5/KyberHintHandler.sol

```diff
@@ -216,84 +247,103 @@ contract KyberHintParser is IKyberHint, Utils {
         bytes memory hint,
         TradeHint memory tradeHint,
         uint indexToContinueFrom,
-        bool isTokenToEth
+        bool isTokenToEth,
+        bool isExternal
     )
         internal
         view
     {
         bytes memory opcode = hint.slice(indexToContinueFrom, 1);
         bytes32 opcodeHash = keccak256(opcode);
```

Comment:
I am not sure I understand why using keccak256?
is it for saving gas?

anyway.
please make sure the opcode compare is most gas efficient.
I am not sure it is right now.

What would be gas consumption if just take opcode

and later do: if(opcode == MASK_IN_OPCODE)
instead of keccak256 for both?
seems to me extra code is running here.
unless u found that this method is more gas efficient.
see comment below on same issue.

Reply:
Can't compare of types bytes memory and bytes memory. Added another set of constants which is the keccak256 of the opcode.